### ISSUE TITLE
PerformanceSensitive: Determine IDs for diagnostics

### ISSUE THREAD

**Issue by edespong** [dotnet/roslyn-analyzers#2259]
Follow-up issue from review of https://github.com/dotnet/roslyn-analyzers/pull/2214#issuecomment-478129830

> > What should the DiagnosticDescriptor ids be? There are currently build errors due the category being Performance, but the current ids are not in the CA1800-CA1899 range
> 
> For now, let's suppress the warning that gets reported for the use of `HAA*`, and file a follow-up issue to determine if these are the final IDs we want to use.

**Comment by Evangelink** [comment]
@mavasani I guess you need to have some internal discussion for this, don't you?

**Comment by mavasani** [comment]
Yes, @sharwell and @genlu should make a call here before we publish this package on NuGet - currently this package is only being published on myget and is an internal-only package.

**Comment by Youssef1313** [comment]
@mavasani It's on NuGet now, was that done accidentally? https://www.nuget.org/packages/Microsoft.CodeAnalysis.PerformanceSensitiveAnalyzers

**Comment by mavasani** [comment]
Yes it should not have been published. @jmarolf do you know when it got published?

**Comment by jmarolf** [comment]
I don't know. Let me investigate

**Comment by Evangelink** [comment]
According to the tooltip on nuget, it was published on 2021/03/02 at 06:10:35Z.
